gradle.properties

```diff
@@ -5,15 +5,17 @@ kotlin.code.style=official
 
 ### Publishing
 
+# NOTE: Can be set back to kyonifer after merging 
+
 # Set up your maven coordinates here, artifactId is defined per project.
-version = 0.12
-group = com.kyonifer
+version = 0.12.1
+group = com.core
```

Comment:
Presumably this should read `com.hellocore`, as that is the domain you own correct? We could also just namespace this at com.kyonifer still, either way works for me.